### PR TITLE
Accuracy Analysis

### PR DESCRIPTION
**Pull request by schwaaweb**
This moves the accuracy reporting functionality that was under SentenceClassifier.test into AccuracyAnalysis.analyze, which takes advantage of the sklearn.metrics library.

This also adds a classifier report to the previous functionality.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@
 **/uploads/*.txt
 *.pyc
 *.env
+package-lock.json
```
Comment (navjotts) [inline .gitignore]: @schwaaweb : removing this as package-lock.json is supposed to be checked in (though I m wondering why it didn't show up for others)

**Comment by navjotts** [comment]
Okay this PR is ready to merge - I m merging it
but for review purposes, @astrebeck @tzunun please review if interested
